src/shared/parameter_v2/examples/generated_dynamic_parameters.hpp

```diff
@@ -0,0 +1,351 @@
+#pragma once
+#include <iostream>
+#include <boost/program_options.hpp>
+#include "software/parameter/config.h"
+#include "software/parameter/numeric_parameter.h"
+#include "software/parameter/enumerated_parameter.h"
+
+class foo;
+class example;
+
```

Comment:
if this is to be merged, a brief comment explaining the document would be helpful

Reply:
would be good to have a rough idea of what the intended input was, i.e. what configs include other configs, etc.

Reply:
its from the `shared/parameter_v2/config_definitions/test` folder

Reply:
Yeah once this is actually generated, not sure if this is something people should be reading directly vs just the config files

Reply:
Can we just add a quick comment at the top since it isn't obvious

Reply:
Done